site/js/ga-bootstrap.js

```diff
@@ -1,37 +1,37 @@
 /* bootstrap GA tracking code */
-try{
+// try{
 
- var _gaq = _gaq || [];
-  _gaq.push(['_setAccount', 'UA-1001800-1']);
-  _gaq.push(['_trackPageview']);
-  (function() {
-  var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-  ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-  var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-  })();
+//  var _gaq = _gaq || [];
```

Comment:
Can this code be deleted rather than commented out?

Reply:
Sure - we can delete that file altogether